product_configurator_refactor/models/product_attribute.py

```diff
@@ -18,9 +18,15 @@ def name_get(self):
             name = value.name
             if self._context.get('show_price'):
                 # Get Currency symbol
-                currency_id = value.product_id.product_tmpl_id.company_id.currency_id
+                company_currency = self.env.user.company_id.currency_id
+                product_currency = value.product_id.product_tmpl_id.company_id.currency_id
+                val = value.product_id.lst_price
+                if product_currency != company_currency:
+                    val = product_currency.compute(value.product_id.lst_price,
+                                                   company_currency)
                 name = '%s (%s%s)' % (
-                    value.name, currency_id and currency_id.symbol or '$',
+                    value.name,
+                    company_currency and company_currency.symbol or '$',
```

Comment:
@smangukiya The Euro symbol should be after the amount. The currency definition has a field for this:

![res_currency](https://user-images.githubusercontent.com/939952/30545149-6b8870c2-9c4e-11e7-9f30-b2bd22067919.png)

Reply:
@max3903 : **Done!**